example/lib/main.dart

```diff
@@ -12,7 +13,7 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  Map<String, dynamic> _headers;
+  Map<String, String>? _headers;
```

Comment:
Is changing `dynamic` to `String` required as part of the null safety migration?

Reply:
`dynamic` is like `Object?` in nnbd, but our map does not contains null value

Reply:
I see that makes sense